src/duration.js

```diff
@@ -607,6 +607,18 @@ export default class Duration {
     return this.toISO();
   }
 
+  /**
+   * Returns a string representation of this Duration appropriate for the REPL.
+   * @return {string}
+   */
+  [Symbol.for("nodejs.util.inspect.custom")]() {
+    if (this.isValid) {
+      return `Duration { values: ${JSON.stringify(this.values)} }`;
```

Comment:
Author note: I've used JSON.stringify for simplificy but it makes Duration's format different from the others. If maintainers prefer, I could change to something like:

```suggestion
      return `Duration { ${Object.entires(this.values).map(([key, val]) => `${key}: ${val}`).join(', ')} }`;
```

Reply:
I think the json stringify is probably right